src/symbols.js

```diff
@@ -732,13 +731,53 @@ for (let i = 0; i < textSymbols.length; i++) {
 }
 
 // All of these are textords in text mode, and mathords in math mode
-const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+const letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
 for (let i = 0; i < letters.length; i++) {
     const ch = letters.charAt(i);
     defineSymbol(math, main, mathord, ch, ch);
     defineSymbol(text, main, textord, ch, ch);
 }
 
+// The next loop loads wide (surrogate pair) characters.
+// We support some letters in the Unicode range U+1D400 to U+1D7FF,
+// Mathematical Alphanumeric Symbols.
+// Some editors do not deal well with surrogate pairs. So don't write the
+// string into this file. Instead, create the string from the surrogate pair.
+let wideChar = "";
+for (let i = 0; i < letters.length; i++) {
+    const ch = letters.charAt(i);
+
+    // The hex numbers in the next line are a surrogate pair.
+    // 0xD835 is the high surrogate for all letters in the range we support.
+    // 0xDC00 is the low surrogate for bold A.
+    wideChar = String.fromCharCode(0xD835, 0xDC00 + i);  // A-Z a-z bold
+    defineSymbol(math, main, mathord, ch, wideChar);
+    defineSymbol(text, main, textord, ch, wideChar);
+
+    // We omit italic. Math mode A-Za-z are italic by default.
```

Comment:
Correct. Any characters from that range will be unsupported. I couldn't think of a reason why anyone would want to use characters from that range, since the default is already italic.

But perhaps it's just that my imagination isn't broad enough. Let me know if you want to include that range in this PR.

Reply:
People might be copy/pasting from other software where this range is being used for italics.

Reply:
I went ahead and added those characters in the most recent commit.